torch/storage.py

```diff
@@ -873,9 +873,9 @@ def untyped(self):
         return self._untyped_storage
 
     def _new_wrapped_storage(self, untyped_storage) -> Self:
-        assert type(untyped_storage) == torch.UntypedStorage
+        assert isinstance(untyped_storage, torch.UntypedStorage)
 
-        if type(self) == TypedStorage:
+        if isinstance(self, TypedStorage):
```

Comment:
Not sure about these two changes... Seems safe, but I have no idea how pybind11 / inductor will like them.